message.exception.dao.default=Произошло исключение в DAO
message.exception.dao.not_found=Объект не найден
